uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Service/Stats/StatsGathererTest.php

```diff
@@ -115,6 +135,10 @@ public function testEntityCountsWillMatchExpectedCount() {
     $this->assertEquals(1, $stats->getEntityCount('assignment'));
     $this->assertEquals(2, $stats->getEntityCount('document'));
     $this->assertEquals(2, $stats->getEntityCount('leaveRequest'));
+
+    $this->assertEquals(2, $stats->getEntityCount('jobRole'));
+    $this->assertEquals($costCenterCount, $stats->getEntityCount('costCenter'));
+    $this->assertEquals(2, $stats->getEntityCount('caseType'));
   }
```

Comment:
Please mention on the PR description that the tests don't cover HR Resources and why

Reply:
Personally, I think that the bigger issue is the fact that we're using the CMS to store data. Besides the problem with the tests, this also introduces some other issues. For example, how can the sample data extension (which is a CiviCRM extension) create sample HR Resources?

Anyway, I don't think this will be fixed anytime soon and the value of adding tests to cover the HR Resources count is low compared to the amount of work that will be necessary. It's ok to not add tests for this.